docker.js

```diff
@@ -170,6 +171,10 @@ const createContainer = async (project, domain) => {
         })
 }
 
+const getStaticFileUrl = async (project, filePath) => {
+    return `http://${project.id}:2880/flowforge/files_/${filePath}`
```

Comment:
```suggestion
const getStaticFileUrl = async (instance, filePath) => {
    return `http://${instance.id}:2880/flowforge/files_/${filePath}`
```

keep with instance naming as in other new functions.